download-translations

```diff
@@ -14,31 +14,42 @@ LANGUAGES=\
 # for testing/debugging:
 #LANGUAGES="es fr hu sv pl xx"
 
+# print error message and exit; prefix problem matcher when in GitHub Actions
+err_msg() {
+  local prefix
+  [ -n "$CI" ] && prefix="::error::"
+  echo $prefix"$*" >&2
+  exit 1
+}
+
+# print warning message; prefix problem matcher when in GitHub Actions
+warn_msg() {
+  local prefix
+  [ -n "$CI" ] && prefix="::warning::"
+  echo $prefix"$*" >&2
+}
+
 # check for 'diff' program
 diff --version 2>/dev/null >/dev/null
 if [ $? -ne 0 ]; then
-  echo "==== You must have the 'diff' program installed for this script ===="
-  exit 1
+  err_msg "==== You must have the 'diff' program installed for this script ===="
```

Comment:
with this you can actually write:
diff --version 2>/dev/null >/dev/null | err_msg "==== You must have the 'diff' program installed for this script ===="

(also below)

Reply:
If you want, we can also omit the "===="

Reply:
Done, reduced some lines.